docs/release-notes.txt

```diff
@@ -8,6 +8,49 @@
  Version 0.13
 **************
 
+Version 0.13.7
+==============
+
+**Release Date:** October 29, 2020
+
+**New Features**
+
+-  Add support for running workloads on spot instances on AWS. Spot instances can be up to 70% cheaper than on-demand instances. If a spot instance is terminated, Determined's built-in fault tolerance means that model training will continue on a different agent automatically. Spot instances can be enabled by setting ``spot: true`` in the :ref:`cluster-configuration`. For more details, see the guide on using :ref:`aws-spot`.
+
+-  Support `MMDetection <https://github.com/open-mmlab/mmdetection>`__, a popular library for object detection, in Determined. MMDetection allows users to easily train state-of-the-art object detection models; with Determined, users can take things one step further with cutting-edge distributed training and hyperparameter tuning to further boost performance. See the `Determined implementation of MMDetection <https://github.com/determined-ai/determined/tree/master/examples/computer_vision/mmdetection_pytorch>`__ for more information on how to get started.
+
+-  WebUI: Allow the experiments list page to be filtered by labels. Selecting more than one label will filter experiments by the intersection of the selected labels.
+
+**Deprecated Features**
+
+-  Deprecate the :ref:`simple <topic-guides_hp-tuning-det_adaptive-simple>` and :ref:`advanced <topic-guides_hp-tuning-det_adaptive-advanced>` adaptive hyperparameter search algorithms. They will be removed in a future release. Both algorithms have been replaced with :ref:`topic-guides_hp-tuning-det_adaptive-asha`, which will give state-of-the-art performance with better scalability.
+
+**Improvements**
+
+-  Documentation: Add a guide for :ref:`setup-eks-cluster` for Determined.
+
+-  Master: Support a minimum instance number for dynamic agents. The master will attempt to scale the cluster to at least the configured value at all times. This is configurable via ``provisioner.min_instances`` in the :ref:`cluster-configuration`. Note that this is likely to increase your cloud instance spend.
```

Comment:
We should explain what this is useful for.

Reply:
Yeah, something along the lines of "Support a minimum instance number for dynamic agents to reduce the response time to increasing workload demand, while waiting for additional agents to spawn and register with the master."

Reply:
"This will reduce the response time for initial workload demand because agents will be ready even when no workloads are scheduled." ?